Fix bug in ifconfig parsing

ljden opened the pull request:
Output of ifconfig on ubuntu 18.04. Note `intet addr:` and `Mask:` do not exist. 
```
enp0s3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 10.0.2.15  netmask 255.255.255.0  broadcast 10.0.2.255
        inet6 fe80::a00:27ff:fe4c:eb1d  prefixlen 64  scopeid 0x20<link>
        ether 08:00:27:4c:eb:1d  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 62  bytes 7089 (7.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

ljden committed:
Fix bug in ifconfig parsing

githubuser0xFFFF commented:
Thank you for this pull request. Does the ifconfig parsing still work on Ubuntu 16.04 after this change?

ljden commented:
Hmm, haven't tested on 16.04, this is pn 18.04 with Melodic

ljden commented:
I can possibly write it so that it takes either format

githubuser0xFFFF commented:
If this is possible, this would be one option. The other option is, to provide two differents files net-statistics-18.04 and net-statistics-16.04 and update the documentation markdown file to tell the user to use the right one.

I also noticed this issue when installing on Lubuntu 18.04 but I also had some additional issues to solve for 18.04. Therefore I thing the best solution would be to have a separate folder with the markdown file and the required files for different Linux versions. Therefore the headline is "URsim 5.10 Installation Guide for Ubuntu 16.04" because it is exactly for 16.04.

ljden commented:
Hmm, that's fair, I might update my fork to 18.04 documentation and keep it separate

githubuser0xFFFF commented:
Thank you for your pull request, I hope, I can create a 18.04 documentation within the next days

ljden committed:
Update README.md

Make some minor modifications. Still need to flesh these out and correct mistakes

ljden committed:
Minor updates

ljden committed:
Remove notice at top of README about multiple versions

ljden committed:
Minor updates
